### PR TITLE
Media picker

### PR DESCRIPTION
**Pull request by alinaboghiu**
## What does this change?

Introduces the ?dcr flag and picker for audio/video content, as well as logging that will allow us to track what is rendered in DCR and what in Frontend. 

All logic is defaulting to rendering in frontend for now, until we implement these article type in DCR.

This code is by now a very standard approach, following the Article, Fronts, [TagPages](https://github.com/guardian/frontend/pull/26149), Interactives and [ImageContent](https://github.com/guardian/frontend/pull/26169) pickers. 

## Why? 

This logic is required to allow us to start building Audio and Video articles in DCR. It also allows us to create dashboards to track the progress of the DCR migration. 

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: applications/app/services/dotcomrendering/MediaPicker.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    false
+  }
+
+  def getTier(
+      mediaPage: MediaPage,
+  )(implicit
+      request: RequestHeader,
+  ): RenderType = {
+
+    // defaulting to false until we are ready to release and create a 50% test
```
Comment (ioannakok) [inline applications/app/services/dotcomrendering/MediaPicker.scala:29]: Are we gonna go straight to 50%?

**Review by ioannakok — APPROVED** [review]
Looks great! Just a non-blocking comment out of curiosity!

File: applications/app/services/dotcomrendering/MediaPicker.scala (hunk 1 above)
Comment (alinaboghiu) [reply applications/app/services/dotcomrendering/MediaPicker.scala:29]: Ha no, 0% first, then we'll see

Comment (alinaboghiu) [inline applications/app/services/dotcomrendering/MediaPicker.scala:29]:
```suggestion
    // defaulting to false until we are ready to release and create a 0% test
```

Comment (alinaboghiu) [reply applications/app/services/dotcomrendering/MediaPicker.scala]: fixed!

**Pull request merged**